src/neuroconv/datainterfaces/text/timeintervalsinterface.py

```diff
@@ -47,7 +48,7 @@ def get_metadata(self) -> dict:
         return metadata
 
     def get_metadata_schema(self) -> dict:
-        fpath = os.path.join(os.path.split(__file__)[0], "timeintervals.schema.json")
+        fpath = Path(__file__).parent / "../../schemas/timeintervals_schema.json"
```

Comment:
Though it might look awkward, I highly recommend letting `pathlib.Path` do all the work for navigation and separators to handle any potential cross-platform differences

```suggestion
        fpath = Path(__file__).parent.parent.parent / "schemas" / "timeintervals_schema.json"
```